### ISSUE TITLE
Cannot generate iOS App archive in xcode

### ISSUE THREAD

**Issue by cordovapolymer** [Taracque/cordova-plugin-braintree#22]
Hi,
When building a archive on IOS, IOS App archive is not generated, instead a Generic Xcode archive is generated.

I tried everything from http://stackoverflow.com/a/13620628 and http://stackoverflow.com/questions/20369290/xcode-is-creating-generic-xcode-archive-instead-of-ios-app-archive

**Comment by Taracque** [comment]
What is the question? Do you want to make an Archive to upload it to the App Store? User Product / Archive command. If that fails at least post the Error message.

**Comment by cordovapolymer** [comment]
When I'm doing Product->Archive a Generic Xcode archive is generated instead of a IOS App archive

**Comment by Taracque** [comment]
I guess it depends on the project setup. Why you think that is connected to Braintree plugin?
Verify the Base SDK setting (should be something iOS), also supported platform should be iOS. macOS, tvOS and watchOS Deployment Target should be empty.

**Comment by cordovapolymer** [comment]
@Taracque because when I remove it, the IOS APP archive is generated.

**Comment by Taracque** [comment]
Which version of cordova and XCode is used?
Can you upload an example project?

**Comment by cordovapolymer** [comment]
@Taracque 
Latest versions, Xcode 8.3.2, cordova 6.50, cordova ios platform 4.4.0. Been tried all different options today.. when I remove the plugin the IOS archive is built correctly.
It will be difficult for me to upload a sample project now, because I will have to create it from scratch.

**Comment by Taracque** [comment]
Can you please try to create a new cordova project, add the plugin and try to build and archive.

**Comment by cordovapolymer** [comment]
@Taracque 
Found the issue! I've been building an archive without trying it emulator because everything went well before installing the plugin. There's no error when building for Generic iOS device, but when I tried building for iOS simulator the following error occured:
`BraintreeDropIn/BraintreeDropIn.h not found`

I tried reinstalling the plugin and ios platform as recommended in the readme.
